src/matcalc/utils.py

```diff
@@ -53,13 +52,78 @@
 except Exception:  # noqa: BLE001
     warnings.warn("Unable to get pre-trained MatGL universal calculators.", stacklevel=1)
 
-# Provide simple aliases for some common models. The key in MODEL_ALIASES must be lower case.
-MODEL_ALIASES = {
-    "tensornet": "TensorNet-MatPES-PBE-v2025.1-PES",
-    "m3gnet": "M3GNet-MatPES-PBE-v2025.1-PES",
-    "chgnet": "CHGNet-MatPES-PBE-2025.2.10-2.7M-PES",
-    "pbe": "TensorNet-MatPES-PBE-v2025.1-PES",
-    "r2scan": "TensorNet-MatPES-r2SCAN-v2025.1-PES",
+# Different backend libraries (MatGL, MACE, GRACE, etc.) use
+# inconsistent or non-canonical model names (e.g., "small-omat-0",
+# "TensorNet-MatPES-PBE-v2025.1-PES", "GRACE-2L-OMAT-medium-base").
+
+# Users and developers are encouraged to use the model naming convention below.
+# MatCalc Model Naming Convention (Unified model ID format):
+# <Model>-<Dataset>-<Functional>-<Version>-<Size>
+
+# Examples:
+# TensorNet-MatPES-r2SCAN-v2025.1-S
+# MACE-MP-PBE-0-M
+# GRACE-OMAT-PBE-0-L
+
+# This table maps such identifiers into the backend model names that
+# each library expects and will be gradually expanded as new models
+# are released.
+
+# Keys use proper canonical capitalization.
+# Values are the actual model names passed to the backend libraries.
+ID_TO_NAME = {
+    "TensorNet-MatPES-PBE-v2025.1-S": "TensorNet-MatPES-PBE-v2025.1-PES",
+    "TensorNet-MatPES-r2SCAN-v2025.1-S": "TensorNet-MatPES-r2SCAN-v2025.1-PES",
+    "M3GNet-MatPES-PBE-v2025.1-S": "M3GNet-MatPES-PBE-v2025.1-PES",
+    "M3GNet-MatPES-r2SCAN-v2025.1-S": "M3GNet-MatPES-r2SCAN-v2025.1-PES",
+    "CHGNet-MatPES-PBE-v2025.2-M": "CHGNet-MatPES-PBE-2025.2.10-2.7M-PES",
+    "CHGNet-MatPES-r2SCAN-v2025.2-M": "CHGNet-MatPES-r2SCAN-2025.2.10-2.7M-PES",
+    "MACE-MP-PBE-0-S": "small",
```

Comment:
As I mentioned in my previous review, the mapping should not be to MACE-MP-PBE-0-S to small. What does small even mean? This is the argument to the MACE init method. In that case, you should parse "MACE-MP-PBE-0-S" to note that the last letter is S and use small as the input.

Reply:
Even if you want to do mapping, the MACE specific mappings can be within the mace loading part of the code. It should not be that we create chaos on the general ID to name mapping for one code. Anything that is specific to one model family should be isolated within that model family itself.